docs/guide/images.md

```diff
@@ -398,3 +398,170 @@ For an ImageAsset Field to be reused you have to publish it. Only then is notifi
 After establishing media library you can create object Relations between the main Content item and the image Content item being used by it.
 
 To learn more about ImageAsset Field Type and its customization see [Field Type Reference](../api/field_type_reference.md#imageasset-field-type).
+
+## Handling SVG images
+
+Currently, Ibexa DXP does not allow you to store SVG images using `Image` or `ImageAsset` FieldType. Until the full support for this MIME type is in place, you can work things around by relying on `File` FieldType and implementing custom extension which would enable you to display/download files in your templates.
```

Comment:
```suggestion
Currently, [[= product_name =]] does not allow you to store SVG images by using the Image or ImageAsset Field Type. 
Until the full support for this MIME type is in place, you can work things around by relying on the File Field Type.
You must implement a custom extension that lets you display and download files in your templates.
```

Reply:
I am fine with that except for splitting into two sentences: one needs to rely on Field Type **and** implement custom Twig extension. These are two steps explained below.